### PR TITLE
add import/export bucket metadata command

### PR DESCRIPTION
**Pull request by poornas**

### CODE REVIEW

File: cmd/admin-cluster-bucket-export.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Download all bucket metadata for a bucket into zip file.
+     {{.Prompt}} {{.HelpName}} myminio/bucket
+
+  2. Download metadata of all buckets in cluster into zip file.
+	 {{.Prompt}} {{.HelpName}} myminio
```
Comment (harshavardhana) [inline cmd/admin-cluster-bucket-export.go:58]: indentation is off here

File: cmd/admin-cluster-bucket-export.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/colorjson"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/pkg/console"
+)
+
+var adminClusterBucketExportCmd = cli.Command{
+	Name:            "export",
+	Usage:           "exports bucket metadata to zip file",
```
Comment (harshavardhana) [inline cmd/admin-cluster-bucket-export.go:38]:
```suggestion
	Usage:           "backup bucket metadata to a zip file",
```

File: go.mod [hunk 3]
```diff
@@ -19,7 +19,7 @@ require (
 	github.com/minio/cli v1.22.0
 	github.com/minio/colorjson v1.0.2
 	github.com/minio/filepath v1.0.0
-	github.com/minio/madmin-go v1.3.13
+	github.com/minio/madmin-go v1.3.17
```
Comment (harshavardhana) [inline go.mod:22]:
```suggestion
	github.com/minio/madmin-go v1.3.19
```

File: cmd/admin-cluster-bucket-import.go [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+	"github.com/klauspost/compress/zip"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/pkg/console"
+)
+
+var adminClusterBucketImportCmd = cli.Command{
+	Name:            "import",
+	Usage:           "imports bucket metadata from zipped file",
```
Comment (harshavardhana) [inline cmd/admin-cluster-bucket-import.go:35]:
```suggestion
	Usage:           "restore bucket metadata from a zip file",
```

File: cmd/auto-complete.go [hunk 5]
```diff
@@ -398,6 +398,9 @@ var completeCmds = map[string]complete.Predictor{
 	"/admin/replicate/status": aliasCompleter,
 	"/admin/replicate/remove": aliasCompleter,
 
+	"/admin/cluster/bucket/export": aliasCompleter,
+	"/admin/cluster/bucket/import": aliasCompleter,
```
Comment (klauspost) [inline cmd/auto-complete.go:402]: Why isn't this in `mc admin bucket ...`? Since we are dealing with one bucket, I would expect it to be there?

**Review by klauspost — APPROVED** [review]
Code LGTM with proposed changes. Just a Q about commandline...

File: cmd/auto-complete.go (hunk 5 above)
Comment (poornas) [reply cmd/auto-complete.go:402]: it logically belongs with the `mc admin bucket..` commands - however the idea was to move support tools to backup/import  cluster data under `mc admin cluster..`

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**